source/Modules/Controller/information.js

```diff
@@ -28,13 +28,18 @@ export class InformationModule extends ControllerModuleBase {
     ];
   }
 
-  async #internalRequestBalance(accountId, callback) {
-    const getBalance = getKeyringHandler(HANDLER_TYPES.GET_BALANCE, this.keyring);
-    const icpBalance = await getBalance(accountId);
-    if (icpBalance.error) {
-      callback(ERRORS.SERVER_ERROR(icpBalance.error), null);
+  async #internalRequestBalance(subaccount, callback, portConfig) {
+    if (subaccount && Number.isNaN(parseInt(subaccount, 10))) {
```

Comment:
```suggestion
    if (subaccount) {
```
since we have uuids now